src/Umbraco.Web.Common/ActionsResults/UmbracoProblemResult.cs

```diff
@@ -1,8 +1,9 @@
-﻿using System.Net;
+using System.Net;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Umbraco.Cms.Web.Common.ActionsResults
 {
+    // TODO: What is the purpose of this? Doesn't seem to add any benefit
```

Comment:
The only purpose is to ensure the result is the expected format for the frontend (the `new {Message = message}` part).